cmd/system-probe/modules/gpu.go

```diff
@@ -24,24 +24,26 @@ import (
 )
 
 var _ module.Module = &GPUMonitoringModule{}
-var gpuMonitoringConfigNamespaces = []string{gpu.GPUConfigNS}
+var gpuMonitoringConfigNamespaces = []string{gpu.GPUNS}
 
 // GPUMonitoring Factory
 var GPUMonitoring = module.Factory{
 	Name:             config.GPUMonitoringModule,
 	ConfigNamespaces: gpuMonitoringConfigNamespaces,
 	Fn: func(_ *sysconfigtypes.Config, deps module.FactoryDependencies) (module.Module, error) {
+
+		c := gpu.NewConfig()
 		probeDeps := gpu.ProbeDependencies{
 			Telemetry: deps.Telemetry,
-			NvmlLib:   nvml.New(),
+			NvmlLib:   nvml.New(nvml.WithLibraryPath(c.NVMLLibraryPath)),
```

Comment:
I would add a comment here specifying that NVML automatically [will set a default path](https://github.com/NVIDIA/go-nvml/blob/150a069935f8d725c37354faa051e3723e6444c0/pkg/nvml/lib.go#L77-L79) if the argument is an empty string.